src/test/ui/consts/miri_unleashed/enum_discriminants.rs

```diff
@@ -3,6 +3,7 @@
 
 // miri unleashed warnings are not useful and change frequently, so they are silenced above.
```

Comment:
What happens if you remove -Zunleash with the new feature enabled?

Reply:
Good point. This should no longer require the flag.